### PR TITLE
Add debug logging configuration and env option for eventhandler

### LINKED ISSUE #42423: event handler chart should have debug mode
For example, the event handler can only enable debug mode via the `-d`/`--debug` command line argument. There isn't currently a way to set/override this:

https://github.com/gravitational/teleport-plugins/blob/master/charts/event-handler/templates/deployment.yaml#L33-L37

A first-class setting in values.yaml that enables debug mode would be fantastic.

Other plugins seem to have a `[log]` section in their `toml` config file that sets output and severity. I did a spot check on the slack chart, and it does have a `log` section in its values that renders into its spot in the config file.

### PR DESCRIPTION
**Pull request by stevenGravy**
fixes https://github.com/gravitational/teleport/issues/42423

Provides a debug setting within the Teleport Event handler toml config and the environment variable `FDFWD_DEBUG`.  Also enables this to be set within the helm chart.

```toml
storage = "./storage"
timeout = "10s"
batch = 20
debug = true
```

Will follow up with docs post this merging.

changelog: Debug setting available for event-handler via configuration or FDFWD_DEBUG environment variable.

### CODE REVIEW

File: integrations/event-handler/cli.go [hunk 1]
```diff
@@ -56,6 +56,9 @@ type TeleportConfig struct {
 	// TeleportIdentityFile is a path to Teleport identity file
 	TeleportIdentityFile string `help:"Teleport identity file" type:"existingfile" name:"teleport-identity" env:"FDFWD_TELEPORT_IDENTITY"`
 
+	// TeleportDebugEnabled allows for debug logging
+	TeleporDebugEnabled bool `help:"Configures debug logging on" name:"teleport-debug" env:"FDFWD_DEBUG"`
```
[the remark this answers is not in the document]
Comment (stevenGravy) [reply integrations/event-handler/cli.go]: @tigrato please take a look at the updates. The variable has been moved to the top level with the name `debugLogging`.

[the remark this answers is not in the document]
Comment (stevenGravy) [reply integrations/event-handler/cli.go]: @tigrato please take a look a the updates when you can.

**Review by tigrato — APPROVED** [review]
@stevenGravy can you please test my latest changes?

**Comment by stevenGravy** [comment]
> @stevenGravy can you please test my latest changes?

Yes, looks good!

File: integrations/event-handler/cli.go [hunk 2]
```diff
@@ -214,7 +214,7 @@ type CLI struct {
 	Config kong.ConfigFlag `help:"Path to TOML configuration file" optional:"true" short:"c" type:"existingfile" env:"FDFWD_CONFIG"`
 
 	// Debug is a debug logging mode flag
-	Debug bool `help:"Debug logging" short:"d"`
+	Debug bool `help:"Debug logging" short:"d" env:"FDFWD_DEBUG"`
```
Comment (espadolini) [inline integrations/event-handler/cli.go:217]: Why not use `TELEPORT_DEBUG` like all of our other binaries?

Comment (stevenGravy) [reply integrations/event-handler/cli.go:217]: This matches with the other environment variable approaches. https://github.com/gravitational/teleport/tree/master/integrations/event-handler

File: integrations/event-handler/testdata/config-debug.toml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+key = "testdata/fake-file"
+ca = "testdata/fake-file"
+url = "https://localhost:8888/test.log"
+session-url = "https://localhost:8888/session"
+
+[teleport]
+addr = "localhost:3025"
+identity = "testdata/fake-file"
+refresh.enabled = true
+refresh.interval = "2m"
```
Comment (espadolini) [inline integrations/event-handler/testdata/config-debug.toml:17]:
```suggestion
refresh.interval = "2m"

```

File: integrations/event-handler/main.go [hunk 4]
```diff
@@ -83,8 +79,18 @@ func main() {
...
+	err := logger.Setup(logger.Config{Severity: "debug", Output: "stderr"})
+	if err != nil {
+		fmt.Println(trace.DebugReport(err))
+		os.Exit(-1)
+	}
+}
+
 // start spawns the main process
 func start() error {
+
```
Comment (espadolini) [inline integrations/event-handler/main.go:93]:
```suggestion
```

**Review by espadolini — APPROVED** [review]

**Pull request merged**